biomage/rotate_ci/rotate_ci.py

```diff
@@ -164,6 +164,7 @@ def update_github_secrets(keys, token, org):
         access_key_id = encrypt(ci_keys["key"], access_key_id)
         secret_access_key = encrypt(ci_keys["key"], secret_access_key)
         encrypted_token = encrypt(ci_keys["key"], token)
+        aws_region = encrypt(ci_keys["key"], "eu-west-1")
```

Comment:
The aws_region was added as a secret so it needs to be encrypted (unnecessarily).

It's pretty public so it should actually be a variable instead, could add a ticket to perform that change.